### PR TITLE
added travis and sbt files

### PR DESCRIPTION
**Pull request by avitaln**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name := "credit-cart-networks"
```
Comment (ohadraz) [inline build.sbt]: cart??? :)

File: build.sbt [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  if (version.value.trim.endsWith("SNAPSHOT"))
+    "snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
+  else
+    "releases" at "https://oss.sonatype.org/service/local/staging/deploy/maven2")
+
+startYear := Some(2017)
+organizationHomepage := Some(url("https://github.com/wix"))
+developers := Nil
+scmInfo := Some(ScmInfo(
+  browseUrl = url("https://github.com/wix/credit-cart-networks.git"),
```
Comment (ohadraz) [inline build.sbt]: card, dude, card. With 'd'

File: build.sbt [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  connection = "scm:git:git@github.com:wix/credit-cart-networks.git"
```
Comment (ohadraz) [inline build.sbt]: cart => card

**Comment by avitaln** [comment]
@ohadraz typos fixed

**Review by ohadraz — APPROVED** [review]

**Pull request merged**